test/build/50_ch-image.bats

```diff
@@ -677,6 +677,96 @@ EOF
     ch-image delete tmpimg:tagged
 }
 
+@test 'ch-image build: multistage with --target' {
```

Comment:
We'll want to test that the behavior matches Docker. I wonder if that will be if we export the images to directories and then look for files, rather than examining output?